### DISCUSSION TITLE
Version error

### DISCUSSION THREAD

**Discussion by BobTheLouis** [EssentialsX/Essentials#4398] [Q&A] [answered]
[FIXED]

Error in version:

You are running a server that does not properly support Bukkit plugins that rely on internal Mojang code. Consider using an Essentials replacement for your server software.
Status determining class:
 !net.minecraft.server.v1_17_R1.MinecraftServer
§6Fetching version information...
§6Unable to check your version! Self-built? Build information:
 §cid:'INVALID' branch:'INVALID' isDev:false§6.

**Comment by Octol1ttle** [comment, marked as answer]
Please try to re-download the latest EssentialsX build from https://essentialsx.net/downloads.html and send `/ess version` again
